src/pages/status/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import StatusDashboard from "@site/src/components/StatusDashboard";
+
+export default function Home() {
+  const { siteConfig } = useDocusaurusContext();
+  return (
+    <Layout
+      title={siteConfig.title}
+      description="Status dashboard for conda-forge"
+    >
```

Comment:
```suggestion
import Layout from "@theme/Layout";
import StatusDashboard from "@site/src/components/StatusDashboard";

export default function Home() {
  return (
    <Layout
      title="Status dashboard"
      description="Status dashboard for conda-forge"
    >
```

The title gets added automatically to the siteConfig one.